package.json

```diff
@@ -25,7 +25,8 @@
     "cy-record": "CYPRESS_RECORD_FIXTURES=true cypress open",
     "pree2e-ci-record": "cross-env NODE_ENV=production node build/server &",
     "e2e-ci": "cross-env CYPRESS_USE_FIXTURES=true cypress run --record  --parallel",
-    "run-e2e": "yarn start-test 'yarn start prod' http://localhost:3000 'CYPRESS_USE_FIXTURES=true cypress run --record  --parallel'",
+    "run-e2e": "start-server-and-test 'DISABLE_CSP=true yarn start' :3000 'CYPRESS_USE_FIXTURES=true cypress run --record --parallel'",
```

Comment:
Måtte ha med et flag `DISABLE_CSP` for å kunne bruke `start-server-and-test` (Den fikk header-overflow av csp headerene våre :facepalm: ), men da slipper vi i alle fall å gjette på hvor mange sekunder det tar å starte opp dev-serveren for å kjøre cypress :smile: